Unused variables

Noticing this when building a project with emnapi. Obviously not a serious issue but would be nice to clean up:

```
../emnapi/packages/emnapi/src/emnapi.c:121:7: warning: unused variable 'r' [-Wunused-variable]
  int r = emscripten_resize_heap(old_size + (size_t) change_in_bytes);
      ^
1 warning generated.
../emnapi/packages/emnapi/src/uv/uv-common.c:23:10: warning: unused variable 'q' [-Wunused-variable]
  QUEUE* q;
         ^
1 warning generated.
../emnapi/packages/emnapi/src/uv/unix/thread.c:97:18: warning: unused variable 'attr_storage' [-Wunused-variable]
  pthread_attr_t attr_storage;
                 ^
```

--- a/packages/emnapi/src/emnapi.c
+++ b/packages/emnapi/src/emnapi.c
@@ -118,7 +118,9 @@ napi_status napi_adjust_external_memory(napi_env env,
   }
 
   size_t old_size = emscripten_get_heap_size();
-  int r = emscripten_resize_heap(old_size + (size_t) change_in_bytes);
+  if (!emscripten_resize_heap(old_size + (size_t) change_in_bytes)) {
+    return napi_set_last_error(env, napi_generic_failure, 0, NULL);
+  }
 
   *adjusted_value = (int64_t) emscripten_get_heap_size();
 

--- a/packages/emnapi/src/uv/unix/thread.c
+++ b/packages/emnapi/src/uv/unix/thread.c
@@ -94,7 +94,6 @@ int uv_thread_create_ex(uv_thread_t* tid,
                         void (*entry)(void *arg),
                         void *arg) {
   int err;
-  pthread_attr_t attr_storage;
 
   /* Used to squelch a -Wcast-function-type warning. */
   union {

--- a/packages/emnapi/src/uv/uv-common.c
+++ b/packages/emnapi/src/uv/uv-common.c
@@ -20,7 +20,7 @@ uv_loop_t* uv_default_loop(void) {
 }
 
 int uv_loop_close(uv_loop_t* loop) {
-  QUEUE* q;
+  // QUEUE* q;
   // uv_handle_t* h;
 #ifndef NDEBUG
   void* saved_data;

--- a/packages/test/cgen.config.js
+++ b/packages/test/cgen.config.js
@@ -2,6 +2,7 @@ module.exports = function (_options, { isDebug, isEmscripten }) {
   const compilerFlags = isEmscripten
     ? [
         // ...(isDebug ? ['-sDISABLE_EXCEPTION_CATCHING=0'] : [])
+        ...(isDebug ? ['-Wall'] : []),
         ...(process.env.MEMORY64 ? ['-sMEMORY64=1'] : [])
       ]
     : []

